src/middlewares/product.coffee

```diff
@@ -11,7 +11,7 @@ module.exports = (req, res, next) ->
   storeId = req.app.get 'store_id'
 
   res.promises.push(
-    api.get "stores/#{storeId}/products"
+    api.get "stores/#{storeId}/products?order_by=position"
```

Comment:
Thanks!

Yes, we should do that, and use real related products too. But I want to keep this PR small and fix the most critical features.

I'll add an issue for it.